trainer/deploy-modal-apps.sh

```diff
@@ -7,7 +7,7 @@ set -e
 cd "$(dirname "$0")"
 
 # Default environment
-ENV="prod"
+ENV="dev"
```

Comment:
How do the prod and dev environments differ on Modal?

Reply:
Theoretically they don't, but when we're testing in dev don't want to be pushing live changes to prod. Once I deploy this PR I'll need to push the updated training code to the Modal prod environment.